sound/soc/codecs/max98373-sdw.c

```diff
@@ -208,7 +208,6 @@ static bool max98373_readable_register(struct device *dev, unsigned int reg)
 static bool max98373_volatile_reg(struct device *dev, unsigned int reg)
 {
 	switch (reg) {
-	case MAX98373_R203E_AMP_PATH_GAIN:
```

Comment:
sorry, I really don't understand this. If this is a hardware register that can change on its own, then it needs to be volatile.

Reply:
It is based on comments "I agree to remove AMP_PATH_GAIN from volatile register list" from Ryans. @ryans-lee , can you explain this ?  thanks!

Reply:
I confirm this register is non-volatile.